arcgis-ios-sdk-samples/Utility Network/Display utility association/README.metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+    "category": "Utility Network",
+    "description": "Create graphics for utility associations in a utility network.",
+    "ignore": false,
+    "images": [
+        "display-utility-association.png"
+    ],
+    "keywords": [
+        "associating",
+        "association",
+        "attachment",
+        "connectivity",
+        "containment",
+        "relationships",
+        "AGSGraphicsOverlay",
+        "AGSUtilityAssociation",
+        "AGSUtilityAssociationType",
+        "AGSUtilityNetwork"
+    ],
+    "redirect_from": [
+    ],
+    "relevant_apis": [
+        "AGSGraphicsOverlay",
+        "AGSUtilityAssociation",
+        "AGSUtilityAssociationType",
+        "AGSUtilityNetwork"
+    ],
+    "snippets": [
+        "DisplayUtilityAssociationViewController.swift"
+    ],
+    "title": "Display utility associations"
```

Comment:
```suggestion
    "title": "Display utility association"
```

Reply:
According to what in the plist, it seems not to be plural.

Reply:
I decided to add the "s" to the sample instead to match the other SDKs.

Reply:
OK then watch out for the folder name and other filenames, etc! 🙏